api_app/analyzers_manager/observable_analyzers/phishstats.py

```diff
@@ -0,0 +1,56 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+import requests
+
+from api_app.analyzers_manager.classes import ObservableAnalyzer
+from api_app.exceptions import AnalyzerRunException
+from tests.mock_utils import MockResponse, if_mock_connections, patch
+
+
+class PhishStats(ObservableAnalyzer):
+    """
+    Analyzer that uses PhishStats API to check if the observable is a phishing site.
+    """
+
+    base_url: str = "https://phishstats.info:2096/api/"
+
+    def __build_phishstats_url(self) -> str:
+        if self.observable_classification == self.ObservableTypes.IP:
+            endpoint = "phishing?_where=(ip,eq,{input})"
+            name = self.observable_name
+        elif self.observable_classification == self.ObservableTypes.URL:
+            endpoint = "phishing?_where=(url,like,~{input}~)&_sort=-id"
+            domain = self.observable_name.split("/")[2]
+            name = domain.split(".")[0]
+        elif self.observable_classification == self.ObservableTypes.DOMAIN:
+            endpoint = "phishing?_where=(url,like,~{input}~)&_sort=-id"
+            name = self.observable_name.split(".")[0]
+        else:
```

Comment:
u did not add code to handle `generic` case.

Reply:
Yea I was having problem on Genric Type

Can u tell me what the request URL would look like?

Reply:
example: https://phishstats.info:2096/api/phishing?_where=(title,like,~bitcoin~)&_sort=-date

Reply:
So
`endpoint = "phishing?_where=(title,like,~{input}~)&_sort=-date"`
and `input = observable_name`
And observable type will be generic ofc

Correct??

Reply:
yes

Reply:
Cool, I'll add it up then!!!